test/system/home_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'application_system_test_case'
+
+class UsersTest < ApplicationSystemTestCase
+  include Devise::Test::IntegrationHelpers
+
+  test 'should render list of all books sorted by author' do
+    sign_in users(:schmidt)
+    visit root_path
+
+    assert_selector 'table.table.all-books'
+
+    within 'table.table.all-books tbody ' do
+      assert_selector 'tr:nth-child(1) td:nth-child(1)', text: 'J.K. Rowling'
+      assert_selector 'tr:nth-child(2) td:nth-child(1)', text: 'J.R.R.Tolkien'
+    end
+  end
+
+  test 'user searches a book by author in Russian' do
```

Comment:
I would suggest adding a couple more tests, testing edge cases as well such as searching for an empty string, searching for a term which also has leading/trailing whitespaces, searching for a term which should not match anything, etc.

Reply:
Added more tests